docs/source/en/model_doc/dinov2.md

```diff
@@ -55,6 +55,18 @@ with torch.no_grad():
 print((last_hidden_states - traced_outputs[0]).abs().max())
 ```
 
+## Resources
+
+A list of official Hugging Face and community (indicated by 🌎) resources to help you get started with DPT.
+
+- Demo notebooks for DINOv2 can be found [here](https://github.com/NielsRogge/Transformers-Tutorials/tree/master/DINOv2).
+
+<PipelineTag pipeline="image-classification"/>
```

Comment:
cc @MKhalusova would be great if this would be automatically added to the docs, for instance for `xxxForImageClassification` models, not sure if possible